### DISCUSSION TITLE
Week 9

### DISCUSSION THREAD

**Discussion by pabpercab1** [ISPP-G5/NexONG#187] [To-Do]
Revisar nuevas versiones del Failure conditions
Es obligatorio ser usuario piloto
Destacar en la presentación de bluejay

**QUE SE ESPERA:**
- Introducción: Killer opener, elevator pich, competidores, anuncio, customer agreement (terminos de servicios, clausulas abusivas, pricing, SLA, incidencias(tiempo y coste), implicaciones en la implementación), resumen de análisis al TCO a 2 años, primero en desarrollo y en producción, github actions, costes esperados, estimaciones a corto, medio y largo plazo.
- Equipo: Estructura, comittement agreement y su estado, no solo el semanal, sino su evolución con gráfica
- Demo: S3, hay que ver el incremento, que varía de una demo a otra? Destacar lo nuevo, con introducción de la demo, mejoras de UI/UX, ser conforme a las regulaciones (cumplimiento de GDPR, que se acepten los terminos, privacidad con HTTPS), changelog con qué ha cambiado
- Retrospectiva mitad de S3: ALM, calidad de código, rendimiento del equpo, matiz con horas y rendimiento (grafica de dos ejes: x - horas, y - rendimiento), evolución, calendario compartido y como facilita la labor, plan de pruebas de aquí al WPL, problemas y riesgos (si vienen de riesgo o no con tabla, como se ha medido, que se ha hecho, el estado de eso...)
- Lecciones aprendidas, reloj del proyecto, plan de gestión de usuarios pilotos, acciones, gestión, por qué el ranking
- Planificación, captura de pantalla con issues de github, y las que se quedan para más adelante (wontfix) trabajo futuro, replanificación si hace falta, pagos, sandbox, registro, api de correo, hacer planificación de PPL (marketing, redes sociales, etc. )(Se puede hacer algo de implementacion)
- Uso de IA con todo lo de alucinaciones
- Landing page

**Comment by marnunrey2** [comment]
Hay que añadir a la presentacion el ***bluejay*** tambien
